src/components/OrderHistory/ImageCarousel.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import { GalleryContainer, ImageContainer, GalleryImage } from './styles';
+
+interface ImageGalleryProps {
+  images: string[]; // Array of image URLs
+}
+
+export default function ImageGallery(props: ImageGalleryProps) {
+  const { images } = props;
+
+  return (
+    <GalleryContainer>
+      {images.map((image, index) => (
+        <ImageContainer key={index}>
```

Comment:
usually you don't use index as key because if they change the index the keys get messed up